src/sysfiles.c

```diff
@@ -84,15 +84,27 @@ typedef void sig_handler_t ( int );
 #include <mach-o/dyld.h>
 #endif
 
+#include <zlib.h>
+
 
 /* utility to check return value of 'write'  */
 ssize_t echoandcheck(int fid, const char *buf, size_t count) {
```

Comment:
(I am not asking you to actually change this, but: wouldn't this have been one of the few cases were one really could have fixed the formatting of the whole function, given that you touch all of it except the first two and last two lines?)